rust/relay/src/main.rs

```diff
@@ -30,11 +30,11 @@ struct Args {
     #[arg(long, env)]
     listen_ip4_addr: Ipv4Addr,
     /// The websocket URL of the portal server to connect to.
+    #[arg(long, env, default_value = "wss://api.firezone.dev/relay/websocket")]
```

Comment:
I'd rather append `relay/websocket` ourselves, although most users won't use that the ones that do will find that less confusing.
The `<component>/websocket` endpoint is a phoenix convention that we didn't pick and we don't know if they will ever change it and break the users' scripts.
However we can expect the user will know the URL of their firezone instance.

Reply:
Done.